sass/base/_structure.scss

```diff
@@ -24,6 +24,12 @@ body {
             overflow: auto;
         }
     }
+
+    &#channel_view {
+        .channel-view {
+            background: var(--center-channel-bg);
```

Comment:
Could `background: var(--center-channel-bg);` go to plain `.channel-view` below around :83?